design/src/smithy/aggregate_shapes.md

```diff
@@ -28,10 +28,11 @@ Smithy `structure` becomes a `struct` in Rust. Backwards compatibility & usabili
   2. All structs are marked `#[non_exhaustive]`
   3. All structs derive `Debug` & `PartialEq`. Structs **do not** derive `Eq` because a `float` member may be added in the future.
   4. Struct fields are public. Public struct fields allow for [split borrows](https://doc.rust-lang.org/nomicon/borrow-splitting.html). When working with output objects this significantly improves ergonomics, especially with optional fields.
-    ```rust,ignore
-     let out = dynamo::ListTablesOutput::new();
-     out.some_field.unwrap(); // <- partial move, impossible with an accessor
-    ```
+    
+```rust,ignore
+let out = dynamo::ListTablesOutput::new();
+out.some_field.unwrap(); // <- partial move, impossible with an accessor
+```
```

Comment:
I think you want to triple-indent this so that it gets actually nested inside the list item.

```markdown
  4. Struct fields are public. Public struct fields allow for [split borrows](https://doc.rust-lang.org/nomicon/borrow-splitting.html). When working with output objects this significantly improves ergonomics, especially with optional fields.

      ```rust,ignore
      let out = dynamo::ListTablesOutput::new();
      out.some_field.unwrap(); // <- partial move, impossible with an accessor
      ```
```

<img width="1051" alt="image" src="https://user-images.githubusercontent.com/15112080/193253703-4945150d-88a1-4848-9549-54fc0fd060ab.png">

Reply:
Also, any idea what that `ignore` is there for? 🤔

Reply:
Yup! Fixed 👌🏻

Reply:
> Also, any idea what that ignore is there for? 🤔

My guess is that the code block was originally copied from some inline documentation in Rust code and the `ignore` was there to signal to `rustdoc` to not treat as a doc test. I can probably remove it (here and in a few other places).